exchanges/mock/recording_test.go

```diff
@@ -3,39 +3,36 @@ package mock
 import (
 	"net/http"
 	"net/url"
-	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/thrasher-corp/gocryptotrader/encoding/json"
 )
 
 func TestGetFilteredHeader(t *testing.T) {
+	items, err := GetExcludedItems()
+	require.NoError(t, err)
+	assert.NotNil(t, items, "GetExcludedItems should not error")
```

Comment:
🚧 This is not what I added as a suggestion. I think it belongs on the line above, and this one should be about returning not-nil.